feathr-impl/src/test/scala/com/linkedin/feathr/offline/SlidingWindowAggIntegTest.scala

```diff
@@ -255,6 +255,69 @@ class SlidingWindowAggIntegTest extends FeathrIntegTest {
     assertEquals(row1f1f1, TestUtils.build1dSparseTensorFDSRow(Array("f1t1"), Array(12.0f)))
   }
 
+
+  /**
+   * test SWA with lateralview parameters
+   */
+  @Test
+  def testLocalAnchorSWAWithLatest: Unit = {
+    val df = runLocalFeatureJoinForTest(
+      joinConfigAsString = """
+                             | settings: {
+                             |  observationDataTimeSettings: {
+                             |     absoluteTimeRange: {
+                             |         startTime: "2018-05-01"
+                             |         endTime: "2018-05-03"
+                             |         timeFormat: "yyyy-MM-dd"
+                             |     }
+                             |  }
+                             |  joinTimeSettings: {
+                             |     timestampColumn: {
+                             |       def: timestamp
+                             |       format: "yyyy-MM-dd"
+                             |     }
+                             |  }
+                             |}
+                             |
+                             |features: [
+                             |   {
+                             |       key: [x],
+                             |       featureList: ["f1"]
+                             |    }
+                             |]
+    """.stripMargin,
+      featureDefAsString = """
+                             |sources: {
+                             |  swaSource: {
+                             |    location: { path: "src/test/resources/slidingWindowAgg/localSWAAnchorTestFeatureData/daily/yyyy/MM/dd/postfixPath/#LATEST" }
```

Comment:
what about multiple `#LATEST` in the path?
"src/test/resources/slidingWindowAgg/localSWAAnchorTestFeatureData/#LATEST/daily/yyyy/MM/dd/postfixPath/#LATEST"